.cirrus.yml

```diff
@@ -9,6 +9,22 @@ env:
   GCP_PROJECT_MAIN: pg-ci-images
   GCP_PROJECT_DEV: pg-ci-images-dev
 
+  # Define IMAGE_NAMES there to use them on depends_on
+  FREEBSD_13: freebsd-13
+  BULLSEYE: bullseye
+  SID: sid
+  SID_NEWKERNEL: sid-newkernel
+  SID_NEWKERNEL_URING: sid-newkernel-uring
+  WINDOWS_CI_VS_2019_VM: windows-ci-vs-2019
+  WINDOWS_CI_VS_MINGW64_VM: windows-ci-mingw64
+  OPENBSD_VANILLA: openbsd-vanilla
+  OPENBSD_POSTGRES: openbsd-postgres
+  NETBSD_VANILLA: netbsd-vanilla
+  NETBSD_POSTGRES: netbsd-postgres
+  LINUX_DEBIAN_BULLSEYE_CI: linux_debian_bullseye_ci
+  WINDOWS_CI_VS_2019_CONTAINER: windows_ci_vs_2019
+  WINDOWS_CI_VS_MINGW64_CONTAINER: windows_ci_mingw64
```

Comment:
I don't really see the point of these variables? They're longer than the names they end up as?

Reply:
I wanted to use these variables for both tasks and the depends_on. When there is a change on the task name, same change should be applied to depends_on. These variables could solve this problem but like you said alias could work, too.

Reply:
But in that case the variable name should be changed as well.  So the indirection doesn't buy us anything.